### DISCUSSION TITLE
Any app with Skip in the App Store?

### DISCUSSION THREAD

**Discussion by NewCoder2023** [skiptools/skip#254] [Q&A] [answered]
Hey,

is there any app, besides the demo apps from skip, in the App Store? 

 I would like to check out an end result :)

Thanks!

**Comment by marcprux** [comment, marked as answer]
We don't track individual apps, but we know there are a number of them. We will be compiling a list in the near future to serve as a reference.

One recent example is Daily Prompt:

Play Store: https://play.google.com/store/apps/details?id=com.dailyprompt.android
App Store: https://apps.apple.com/kr/app/daily-prompt-creative-writing/id1448668946

Another good example (despite being our own demo) is the Skip Showcase app, which provides a side-by-side comparison of all the controls and views used by a Skip app:

Play Store: https://play.google.com/store/apps/details?id=org.appfair.app.Showcase
App Store: https://apps.apple.com/us/app/skip-showcase/id6474885022

**Reply by aabewhite** [reply]
The authors of ServiceM8 also recently mentioned their release to us: 

https://play.google.com/store/apps/details?id=com.servicem8.skip&hl=en_US

**Reply by davidakoontz** [reply]
Hey - I saw and tried out the Showcase App - NICE!
A useful example of what IS Working / ported / etc. for Android.

This has me wondering about the new iOS 18 TabView & Tab structure... new navigation I was thinking of using in my Skip-able App.
Do you hard working Skippers have a roadmap on when you might get around to new iOS 18 stuff?  
I wonder if vendors (such as yourself) get any lead time for new iOS features...  seems unlikely...

**Comment by NewCoder2023** [comment]
Thank you! @marcprux @aabewhite
